.github/workflows/pull-request.yml

```
@@ -181,19 +181,43 @@
+        id: changed-files
+        uses: tj-actions/changed-files@v45
+
       - name: Set up Python
         uses: actions/setup-python@v5
         with:
           python-version: 3.9
 
       - name: Run Python script
         run: |
-          python .github/workflows/eslint_disable_check.py
+          python .github/workflows/code_coverage_disable_check.py --files ${{ steps.changed-files.outputs.all_changed_files }} 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)